fhir-bucket/src/main/java/com/ibm/fhir/bucket/cos/CosClient.java

```diff
@@ -50,6 +58,7 @@
     // Set to false to tell a scan to return early
     private volatile boolean running = true;
     
+    
     static {
         SDKGlobalConfiguration.IAM_ENDPOINT = "https://iam.cloud.ibm.com/oidc/token";
     }
```

Comment:
Do we care about binding this to the IBM Cloud?

Reply:
HMAC is a more s3-compatible way to configure access to COS buckets so I'd suggest that we eventually move to that.  One downside is that its not the "default" in IBM Cloud...users must take explicit action to get HMAC creds.

Reply:
I'm not happy with this either, but it comes from the bulk importer.